### PR TITLE
Add CHANGELOG items for v2.25.0

### PR DESCRIPTION
**Pull request by dataclouder**
Prepare CHANGELOG for v2.25

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,116 @@
-## 2.25.0 (Unreleased)
+## 2.25.0 (July 2, 2024)
+
+### FEATURES
+* Added types `SolutionLandingZone` and `types.SolutionLandingZone` for Solution Add-on Landing Zone configuration ([#665](https://github.com/vmware/go-vcloud-director/pull/665))
+* Added method `DefinedEntity.Refresh` to reload RDE state ([#665](https://github.com/vmware/go-vcloud-director/pull/665))
+* Added `VDCClient` methods `CreateSolutionLandingZone`, `GetAllSolutionLandingZones`,
```
Comment (Didainius) [inline CHANGELOG.md:6]:
```suggestion
* Added `VCDClient` methods `CreateSolutionLandingZone`, `GetAllSolutionLandingZones`,
```

**Review by lvirbalas — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,116 @@
...
+  and `CseWorkerPoolUpdateInput.Autoscaler`, that allows to configure this mechanism on specific worker pools ([#678](https://github.com/vmware/go-vcloud-director/pull/678))
+* Added types `SolutionAddOnInstance` and `types.SolutionAddOnInstance` for Solution Add-on Instance
+  management ([#679](https://github.com/vmware/go-vcloud-director/pull/679))
+* Added `VCDClient` methods `GetAllSolutionAddonInstanceByName`, `GetAllSolutionAddonInstances`,
+  `GetSolutionAddOnInstanceById` ([#679](https://github.com/vmware/go-vcloud-director/pull/679))
+* Added `SolutionAddOn` methods `CreateSolutionAddOnInstance`, `GetAllInstances`,
+  `GetInstanceByName`, `ValidateInputs`, `ConvertInputTypes` ([#679](https://github.com/vmware/go-vcloud-director/pull/679))
+* Added `SolutionAddOnInstance` methods `GetParentSolutionAddOn`, `ReadCreationInputValues`
+  `Delete`, `RdeId`, `Publishing` ([#679](https://github.com/vmware/go-vcloud-director/pull/679))
+* Added methods to create, read, update and delete VDC Templates: `VDCClient.CreateVdcTemplate`, `VDCClient.GetVdcTemplateById`,
```
Comment (Didainius) [inline CHANGELOG.md:47]:
```suggestion
* Added methods to create, read, update and delete VDC Templates: `VCDClient.CreateVdcTemplate`, `VCDClient.GetVdcTemplateById`,
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,116 @@
...
+`VDCClient.GetVdcTemplateByName`, `VdcTemplate.Update` and `VdcTemplate.Delete` ([#686](https://github.com/vmware/go-vcloud-director/pull/686))
+* Added methods to manage the access settings of VDC Templates: `VdcTemplate.SetAccessControl` and  `VdcTemplate.GetAccessControl` ([#686](https://github.com/vmware/go-vcloud-director/pull/686))
+* Added the `VdcTemplate.InstantiateVdcAsync` and `VdcTemplate.InstantiateVdc` methods to instantiate VDC Templates ([#686](https://github.com/vmware/go-vcloud-director/pull/686))
+* Added the `VDCClient.QueryAdminVdcTemplates` and `Org.QueryVdcTemplates` methods to get all VDC Template records ([#686](https://github.com/vmware/go-vcloud-director/pull/686))
```
Comment (Didainius) [inline CHANGELOG.md:51]:
```suggestion
* Added the `VCDClient.QueryAdminVdcTemplates` and `Org.QueryVdcTemplates` methods to get all VDC Template records ([#686](https://github.com/vmware/go-vcloud-director/pull/686))
```

**Review by Didainius — APPROVED** [review]

**Review by adambarreiro — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (dataclouder) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (dataclouder) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 3 above)
Comment (dataclouder) [reply CHANGELOG.md]: fixed

**Pull request merged**